tyk_reverse_proxy_clone.go

```diff
@@ -314,7 +314,7 @@ func (p *ReverseProxy) New(c interface{}, spec *APISpec) (TykResponseHandler, er
 }
 
 func (p *ReverseProxy) ServeHTTP(rw http.ResponseWriter, req *http.Request) *http.Response {
-	return p.WrappedServeHTTP(rw, req, RecordDetail(req))
```

Comment:
This method actually enforces detailed analytics recording, not caching, so we need to call it...